src/crate/theme/rtd/crate/static/css/crateio.css

```diff
@@ -113,14 +113,15 @@ a:hover {
   color: #55d4f5;
 }
 
-ul {
+ul, ol {
   margin-top: 0px;
-  margin-bottom: 10px;
+  margin-bottom: 16px;
+  margin-left: 16px;
```

Comment:
Could we be more specific where this margin is getting added? like `.wrapper-content-right ul, .wrapper-content-right ol { ... }`? Else the mobile menu and the sidebar with the docs navi (mobile) get that margin as well and alters the layout quite significantly